contrib/platform/test/com/sun/jna/platform/win32/AbstractWin32TestSupport.java

```diff
@@ -0,0 +1,45 @@
+/* Copyright (c) 2007 Timothy Wall, All Rights Reserved
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ */
+package com.sun.jna.platform.win32;
+
+import com.sun.jna.platform.AbstractPlatformTestSupport;
+
+/**
+ * @author lgoldstein
+ */
+public abstract class AbstractWin32TestSupport extends AbstractPlatformTestSupport {
```

Comment:
I have no issues with that but _checkBool_ must be defined somewhere - it could be in a separate _Win32TestSupportUtils_, but this is IMO just semantics - moving it from one class (_AbstractWin32TestSupport_) to another (_Win32TestSupportUtils_). Also, please note that the name _assertCallSucceeded_ is aligned with the _jUnit_ assertions - i.e., it indicates that the assertion will *fail* if it is not true, whereas _checkBool_ does not do this.

``` unless you want to rewrite all tests to use this.``` I don't see why this is a requirement. These are *helper* classes (which IMO are _very_ helpful) which did not exist so far. There is no need to retro-fit anything. New tests _may_ be written using them (or not)...